packages/example/utils/locale.ts

```diff
@@ -0,0 +1,51 @@
+import { Locale } from '@rainbow-me/rainbowkit';
+// Detect user locale from browser
+export const getDetectedBrowserLocale = () => {
+  if (typeof window !== 'undefined' && typeof navigator !== 'undefined') {
+    if (navigator.languages?.length) {
+      return navigator.languages[0];
+    }
+
+    if (navigator.language) {
+      return navigator.language;
+    }
+  }
+};
+
+// We will get locale from browser and return the locale which we support
+export const getSupportedLocaleFromBrowser = (locale: string) => {
+  switch (locale) {
+    case 'en-US':
+      return Locale.EN_US;
+
+    case 'es-419':
+      return Locale.ES_419;
+
+    case 'fr-FR':
+      return Locale.FR_FR;
+
+    case 'ja-JP':
+      return Locale.JA_JP;
+
+    case 'pt-BR':
+      return Locale.PT_BR;
+
+    case 'zh-CN':
+      return Locale.ZH_CN;
+
+    case 'id-ID':
```

Comment:
Bunch of `switch` cases based off of browsers `locale`. We could go ahead and change our `Locale` enum to match browsers `locale`, but better to use underscores `_` like `en_US` 🙌

Reply:
Reference this above; think we should bring this into the package to simplify Next.js use